src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public abstract class Figure extends ColorSupplier implements AreaCalculator, Drawer {
+    private double area;
```

Comment:
we don't need this field actually

let's remove it, and its getters and setters as well

Reply:
add